executor/show.go

```diff
@@ -175,6 +175,14 @@ func (e *ShowExec) fetchShowProcessList() error {
 		if len(pi.Info) != 0 {
 			t = uint64(time.Since(pi.Time) / time.Second)
 		}
+
+		var info string
+		if e.Full {
+			info = pi.Info
+		} else {
+			info = fmt.Sprintf("%.100v", pi.Info)
```

Comment:
You mean truncate info field length rather than item counts?

Reply:
yes, truncate the info field, the value 100  is from mysql source code:

```c
sql/sql_const.h:#define PROCESS_LIST_WIDTH 100
```
mysqld_list_processes is defined in sql/sql_show.cc

mysql truncates the info fields byte-wize, but using go's fmt.Sprint above it truncates at the unicode width boudary, but i don't think that would be an issue, do you?

Reply:
ok